plugins/mollie/README.md

```diff
@@ -0,0 +1,37 @@
+FoodsoftMollie
+==============
+
+This project adds support for iDEAL payments using Mollie to Foodsoft.
+
+* Make sure the gem is uncommented in foodsoft's `Gemfile`
+* Enter your Mollie account details in `config/app_config.yml`
+
+```yaml
+  # Mollie payment settings
+  mollie:
+    # API key for account: 1234567, website profile: FooInc
+    api_key: test_1234567890abcdef1234567890abcd
+    # Transaction fee per payment method, fixed rate and/or percentage.
+    #   This is substracted from the amount actually credited to the ordergroup's account balance.
+    fee:
+      # example fees from May 2014 incl. 21% VAT (verify before using!)
+      ideal: 1.20
+      banktransfer: 0.30
+      creditcard: 3.39% + 0.05
+      paypal: 0.18 + 0.35 + 3.4%
```

Comment:
I think this is quite tricky. There is a way to get fee's from the mollie api, but e.g. paypal charges additional fees ... But maybe this is only the exception. 

Maybe this would be a sane config interface: 
* [] Charge ordergroup mollie fee's 
  * [] defaults from mollie-api
  * Input custom fee for  $payment_method

Reply:
I thought a override might be useful if the actual costs differ for some reason (I've read smth about paypal having extra fee's, but I'm not sure). But yeah maybe overkill for the moment.

Via the [/methods endpoint](https://docs.mollie.com/reference/v2/methods-api/get-method#example) it is possible to get mollies fees. And yess it's a bit more complex :o
Though I haven't found a way to get a computed fee from mollie payment result.

Having the option to topup the fee amount would be great. I think I'll add the method and issuer selection again, so the fee can be computed beforehand.